.github/workflows/cqa.yml

```diff
@@ -18,30 +18,24 @@ jobs:
         with:
           python-version: 3.9
 
-      - uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
+#      - uses: actions/cache@v2
+#        with:
+#          path: ~/.cache/pip
+#          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+#          restore-keys: |
+#            ${{ runner.os }}-pip-
```

Comment:
i'm not sure exactly how you intended this to be used since we don't have a `requirements.txt` file, but for the time being it's not effective and is causing a large amount of data (38Mb) to be compressed and stored at the end of the job because the `~/.cache/pip` directory is quite large.
![image](https://user-images.githubusercontent.com/4059210/145049359-3ec35689-c214-434f-8e5c-b0c1f43772c8.png)